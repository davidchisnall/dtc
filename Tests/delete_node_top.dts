/dts-v1/;

/ {
        node1 { 
                test: node3 {};
        };
};

/ {
        test2: node2 { };
};

/delete-node/ &test;
/delete-node/ &test2;

