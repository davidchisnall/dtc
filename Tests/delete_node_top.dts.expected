/dts-v1/;

/  {

	node1 {

	};
};
